models/articles.models.js

```diff
@@ -2,7 +2,29 @@ const db = require('../db/connection');
 
 exports.selectArticleById = (article_id) => {
 	return db
-		.query('SELECT * FROM articles WHERE article_id=$1;', [article_id])
+		.query(`SELECT * FROM articles WHERE article_id=$1;`, [article_id])
+		.then((result) => {
+			return db
+				.query(
+					`
+					ALTER TABLE articles 
```

Comment:
You should not be altering the table to add comment_count, this could lead to two sources of truth, you should use the comments table, and may want to look into how we can JOIN that table to gather the comment_count data. 

You might want to look into COUNT and how we can GROUP BY, to help you with this.